sample/MozillaAllHands_17/tracks

```diff
@@ -0,0 +1,230 @@
+[ 
+    {
+        "id": 455, 
+        "name": "All Hands", 
+        "color": "#2E7D32",  
+        "sessions": [
+			{
+                "id": 1001, 
+                "title": "Registration and Information Desk"
+            }
+		], 
+        "track_image_url": ""
+    },
+	{
+        "id": 456, 
+        "name": "Meals", 
+        "color": "#283593",  
+        "sessions": [
+			{
+                "id": 1002, 
+                "title": "Welcome to San Francisco Reception"
+            },
+			{
+                "id": 1003, 
+                "title": "Breakfast"
+            },
+			{
+                "id": 1004, 
+                "title": "Lunch"
+            },
+			{
+                "id": 1005, 
+                "title": "Breakfast"
+            },
+			{
+                "id": 1006, 
+                "title": "Breakfast"
+            },
+			{
+                "id": 1007, 
+                "title": "Breakfast"
+            },
+			{
+                "id": 1008, 
+                "title": "Lunch"
+            },
+			{
+                "id": 1009, 
+                "title": "Lunch"
+            },
+			{
+                "id": 1010, 
+                "title": "Lunch"
+            }
+		], 
+        "track_image_url": ""
+    },
+	{
+        "id": 457, 
+        "name": "Firefox", 
+        "color": "#37474F",  
+        "sessions": [
+			{
+                "id": 1011, 
+                "title": "Taskcluster 2nd Half 2017 Planning Meeting"
+            },
+			{
+                "id": 1012, 
+                "title": "Taskcluster Migration and Support"
+            },
+			{
+                "id": 1013, 
+                "title": "Release Engineering Retrospective"
+            },
+			{
+                "id": 1014, 
+                "title": "Test Pilot Field Research Findings and Recommendations"
+            },
+			{
+                "id": 1015, 
+                "title": "Q&A Session with Nick Nguyen and NMX Team"
+            },
+			{
+                "id": 1016, 
+                "title": "Firefox Product & Program/Project Management Homeroom"
+            },
+			{
+                "id": 1017, 
+                "title": "Firefox Product & Program/Project Management Homeroom"
+            },
+			{
```

Comment:
formatting the JSON is a good practice and it won't take much time. Just run it through one of the online formatters once.